src/content/docs/dev/general-message-passing/debug/error-debugging.mdx

```diff
@@ -6,7 +6,7 @@ There are two reasons that executions fail:
 
 ### 1. Insufficient gas to execute the transaction
 
-If the prepaid gas is insufficient, you might see `NOT ENOUGH GAS` or other messages indicating the submitted gas is too low to execute, such as `Insufficient gas for executing the transaction`, `intrinsic gas too low`, etc.
+If the prepaid gas is insufficient, for example, this [transaction](https://axelarscan.io/gmp/0x1776026b406ab2aaaedeae83f4a19fc8809000cc8a3b2abd8716587a29460839-1), you might see `NOT ENOUGH GAS` or other messages indicating the submitted gas is too low to execute, such as `Insufficient gas for executing the transaction`, `intrinsic gas too low`, etc.
```

Comment:
im unclear on this. for inssuficnet gas shouldnt users be able to top up missing funds? why cant they do that here?

Reply:
Of course, they should via Axelarjs SDK or a button on the UI. Sometimes it happens on the destination chain and needs to be manually done.